frontend/components/InteractiveImage/ImageElements/FlatSolarPanels.tsx

```diff
@@ -0,0 +1,57 @@
+interface Props {
+  top?: string;
+  left?: string;
+  width?: string;
+  dataTestId?: string;
+}
+
+export default function FlatSolarPanels({ top, left, width, dataTestId }: Props) {
+  return (
+    <div
+      className={`flatSolarpanels ${top ? top : ""} ${left ? left : ""} ${width ? width : ""}`}
+      data-testid={dataTestId ? dataTestId : ""}
+    >
+      <img id="flat" src="/imgs/flat.png" className="absolute top-0 left-0 h-full" alt="flat"></img>
+      <div className="position-static">
+        <img
+          id="solarpanel1"
+          data-testid="solarpanel1"
+          src="/imgs/flat_solarpanels_block.png"
+          className="solarpanelBlock absolute top-[46.5%] left-[62.4%] w-[12%]"
+          alt="solarpanel1"
```

Comment:
A common theme in this PR is that the alt texts aren't useful for their intended purpose. These are used if the image doesn't load and by screenreaders for visually impaired visitors. One tip I've heard is:

> Imagine that you’re reading the web page aloud over the phone to someone who needs to understand the page. This should help you decide what (if any) information or function the images have. If they appear to have no informative value and aren’t links or buttons, it’s probably safe to treat them as [decorative](https://www.w3.org/WAI/tutorials/images/decorative/).
>
> [Images → Tips and tricks. Web Accessibility Initiative](https://www.w3.org/WAI/tutorials/images/tips/)

I would suggest that these images are decorative and that we'll provide other information – such as KPIs – which will be more useful to screen readers. If that's the case, the alt text should be an empty string: `alt=""` to indicate that it should not be read. If not, we should find a more descriptive text to use. A screen reader will announce each of these to a visitor, and "solar panel 1, solar panel 2, etc" isn't useful.

Reply:
@antw Thanks so much for the review! And good points on setting the style and changing the alt texts, I fully agree. I'll implement those.